packages/palette/src/elements/Menu/Menu.tsx

```diff
@@ -14,7 +15,7 @@ interface MenuProps {
   children?: React.ReactNode
   onClick?: (event: React.MouseEvent<HTMLElement>) => void
   title?: string
-  width?: number
+  width?: number | string
```

Comment:
Similar to below, can import `WidthProps` type from `styled-system`

Reply:
I'm seeing some type errors when I try and pull in types that have corresponding `ResponsiveValue<ReactText>`. I will dig in a bit more tomorrow.

Reply:
👍 if its weird no worry, basically just `number | string` under the hood